get_location.py

```diff
@@ -0,0 +1,53 @@
+"""Grab location from API based on users input. Currently set up to search for city names based on input only in the United States."""
+""" Currently when looking up by state only returns the state instead of all cities in the state"""
+import requests
+
+
+params = {
+    'city': '',
+    'country': 'United States',
+    'format': 'json'
+}
+
+def get_location(userInput):
+    """Gather all methods into one in order to create location object. 
+    Only needs to call this function to excecute all of them."""
+    try:
+        create_params(userInput)
+        returned_data = request_nominatim()
+        format_data(returned_data)
```

Comment:
You're formatting the data but haven't returned it.

try:
```
def get_location(userInput):
    """Gather all methods into one in order to create location object. 
    Only needs to call this function to excecute all of them."""
    try:
        create_params(userInput)
        returned_data = request_nominatim()
        return(format_data(returned_data))
```

I got overzealous in approving the file. I apologize.

Reply:
The file returns 'actual_cities' now so it can give the UI the location options for the user. The location object will also be made in main because it would be too complicated/weird returning something then reusing it. Unless you have a good way of doing it.